tests/CMakeLists.txt

```diff
@@ -96,3 +98,15 @@ if(AVIF_ENABLE_GTEST)
 else()
     message(STATUS "Most tests are disabled because AVIF_ENABLE_GTEST is OFF.")
 endif()
+
+################################################################################
+# Bash tests
+
+if(AVIF_BUILD_APPS)
```

Comment:
`AVIF_BUILD_APPS` means "build the executables in the apps/ directory".

Note: We can move are_images_equal.cc to the apps/ directory, but we will need to convert it to a C file, and we will also need to move `libavif::testutil::AreImagesEqual` to apps/shared/avifutil.{h,c} and convert it to a C function.

Reply:
I am adding a test to test the app so I indeed need to check for `AVIF_BUILD_APPS`